### ISSUE TITLE
No hostile mobs in world

### ISSUE THREAD

**Issue by miiichael** [SleepyTrousers/EnderZoo#127]
Hello, I added EnderZoo-1.8.9-1.1.3.35 to my server (FTBUnstable18-3.0.36-1.8.9 plus Covers+BDlib, and VeinMiner), and noticed that the only mobs spawning were Owls plus vanilla cows/sheep/pigs/chickens/bats. When I removed EnderZoo from the server, hostile mobs resumed spawning.

I have attached the [server log](https://github.com/SleepyTrousers/EnderZoo/files/177067/fml-server-1.log.txt) from the first run with the mod installed, in case there's anything in it more useful to you than the mod list...

**Comment by CrazyPants** [comment]
Thanks for the report.
I cannot reproduce this at all with just ez. There is nothing in the log and I cant even think of a way I could make this happen.

**Comment by Xanderr** [comment]
To start of, I love your mods!
I did encounter the same problem as miiichael (and some other people at Curse as well). Although I'm no expert, I did do a few observations:

According to JourneyMap, owls are hostile mobs (probably because they attack spiders). Apparently this is true, since they have influence on the hostile mob spawning cap. Their spawning behaviour is like passive mobs however, causing them to never despawn. I verified this by moving away from an owl more than 128 blocks, wait a minute and go back. It was still there.

Furthermore it seems they are spawning in forest-like biomes. My world spawn point was in a forest and a lot of owls spawned there. As those chunks remain loaded even when you're not there, those owls still take up the spawning cap for hostile mobs. This is probably the reason why mobs are barely spawning.

After I moved my world spawn point to a desert, the problem was resolved.. while in the desert. When I move back to a foresty biome, hostile mob spawns were a little better but not as high as they should be.

Hopefully this will help you pinpoint the issue :) Maybe it can be fixed by adding config options to decrease spawn rates of owls and/or make them despawn when you move away too far.
